spec/models/observation_collection_spec.rb

```diff
@@ -1,9 +1,43 @@
 require 'spec_helper'
 
 describe ObservationCollection do
-  describe ".new" do
-    xit "" do
-      # add tests
+  describe ".new_from_array" do
+    it "returns new ObservationCollection" do
```

Comment:
Oh I see what you mean, if `#find_by` works then it responds to `:each`. We rely on the ability to iterate as part of the public API though, it's not just used internally. I'll write a real test to cover iterability.